src/lib/sw_engine/tvgSwRaster.cpp

```diff
@@ -167,7 +167,7 @@ static bool _rasterMaskedRect(SwSurface* surface, const SwBBox& region, uint32_t
         auto cmp = &cbuffer[y * surface->stride];
         for (uint32_t x = 0; x < w; ++x, ++dst, ++cmp) {
             auto tmp = ALPHA_BLEND(color, blendMethod(*cmp));
-            *dst = tmp + ALPHA_BLEND(*dst, _ialpha(tmp));
+            *dst = surface->blender.blend(color, *dst, _alpha(tmp), _ialpha(tmp), tmp);
```

Comment:
This is the file where I need the most help :)

Reply:
Could you please elaborate this "help" comment?